packages/web/src/store/functions.ts

```diff
@@ -9,11 +11,22 @@ export const startBrowserWorker = async () => {
   return browser.worker.start();
 };
 
-// --- Authentication Functions ---
-export const apiLogin = AuthService.login;
-export const apiLogout = AuthService.logout;
+/**
+ * Define our functions - the library will automatically infer all error types!
+ */
+const functions = {
+  apiLogin: AuthService.login,
+  apiLogout: AuthService.logout,
+  apiGetUsers: UserService.getUsers,
+  apiCreateUser: UserService.createUser,
+  apiDeleteUser: UserService.deleteUser,
+  startBrowserWorker: startBrowserWorker,
+} as const;
+
+// Register with the library
+export const functionRegistry = registerFunctions(functions);
```

Comment:
We first register our functions so that we can later infer all of the return types.